dashboard/serializers_test.py

```diff
@@ -129,14 +130,30 @@ def test_full_program_user_serialization(self):
         """
         Tests that full ProgramEnrollment serialization works as expected
         """
+        Profile.objects.filter(pk=self.profile.pk).update(email_optin=False)
+        self.profile.refresh_from_db()
         program = self.program_enrollment.program
         assert UserProgramSearchSerializer.serialize(self.program_enrollment) == {
             'id': program.id,
             'enrollments': list(CachedEnrollment.active_data(self.user, program)),
             'certificates': list(CachedCertificate.active_data(self.user, program)),
             'current_grades': list(CachedCurrentGrade.active_data(self.user, program)),
             'grade_average': 75,
-            'is_learner': True
+            'is_learner': True,
+            'email_optin': False
+        }
+
+        Profile.objects.filter(pk=self.profile.pk).update(email_optin=True)
```

Comment:
Please make a separate test here to separate the `email_optin` true and false cases. You can use the `ddt` library to test true and false. See here for an example: https://github.com/mitodl/micromasters/blob/master/financialaid/api_test.py#L162